documentation/book/cluster-operator.adoc

```diff
@@ -157,7 +157,7 @@ For information about the accepted JSON format, see <<resources_json_config>>.
 `spec.topicOperator`::
 An object representing the topic operator configuration. 
 See the <<topic_operator_json_config>> documentation for further details.
-More info about the topic operator in the related xref:topic-operator-{context}[Topic Operator] documentation page.
+More info about the topic operator in the related xref:deploying-the-topic-operator-using-the-cluster-operator-{context}[Topic Operator] documentation page.
```

Comment:
We want to replace {context} here with 'str' I think. That will be the default context.

Reply:
I don't think this has been edited yet.